### ISSUE TITLE
Bug: 2 Bottom Bars are showed at every start

### ISSUE THREAD

**Issue by ghost** [lovac42/DancingBaloney#7]
At every start, 2 bottom bars are shown like this:

![2020-05-15 23_50_37-Greenshot](https://user-images.githubusercontent.com/65417822/82083231-ff79ed00-9706-11ea-94e3-63b66dace024.jpg)

Until I change the opacity a little bit or click on hide again. Then it gets normal as it should be:

![2020-05-15 23_52_56-Greenshot](https://user-images.githubusercontent.com/65417822/82083618-a8c0e300-9707-11ea-8650-9dc05c929f0e.jpg)

I wish I don't have to tweak it every time on each start.
Thank You.

**Comment by lovac42** [comment]
Yeah it depends on the load time. I used a timer to hide the bottom bar. You can just press the d key to refresh and that'll hide it. I'm afraid I haven't thought of a clean way to work around this issue.
